css/pysangamam.css

```diff
@@ -18,6 +18,7 @@ body, .main {
 
 h1 {
   font-weight: bold;
+  font-size: 80px;
```

Comment:
Which element are you trying to target with this?

Reply:
In Mobil View , the word Pysangamam is out of screen . So I tried to reduce it .